src/app/components/show-user/show-user.component.html

```diff
@@ -1,4 +1,114 @@
-<div *ngFor="let user of store.users | async">
-  <h3>{{ user.attributes.fullName }}</h3>
-  <h4>{{ user.attributes.email }}</h4>
+<!--navbar starts-->
+<div class="container-fluid" *ngIf="users">
```

Comment:
Using `toolbar` and `list` components from `patternfly-ng` instead of directly using the HTML markup will. be better here. Maybe you can look into it in separate PR.

Reply:
yes sure. I'll raise a pr when I'll make view profile component with angular-ng.

Reply:
angular-ng?